hbase-server/src/main/java/org/apache/hadoop/hbase/master/assignment/TransitRegionStateProcedure.java

```diff
@@ -126,6 +144,14 @@
 
   private boolean isSplit;
 
+  private boolean forceRegionRetainment;
+
+  private ServerManager serverManager;
```

Comment:
ServerManager can be gotten from MasterProcedureEnv, so we do not need to store it here.

Reply:
Problem here is that we don't keep any MasterProcedureEnv ref as class attribute.

Reply:
When executing you will always have a MasterProcedureEnv...